src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 300ms;
```

```suggestion
    transition: transform 300ms, color 300ms;
```
And move 300ms to the variable